docs/git.md

```diff
@@ -47,6 +47,18 @@ The following configuration properties are supported for each provider configura
 : *Required only for private SCM servers*
 : SCM API `endpoint` URL e.g. `https://api.github.com` (default: the same as `providers.<provider>.server`).
 
+`providers.<provider>.retryPolicy.delay`
+: Delay when retrying failed requests (default: `500ms`).
+
+`providers.<provider>.retryPolicy.jitter`
+: Jitter value when retrying failed requests (default: `0.25`).
+
+`providers.<provider>.retryPolicy.maxAttempts`
+: Max attempts when retrying failed requests (default: `10`).
+
+`providers.<provider>.retryPolicy.maxDelay`
+: Max delay when retrying failed requests (default: `90s`).
+
```

Comment:
we are starting to have too many of those settings replicated for different components. Wonder if it would not be better having a common one

Reply:
I think that would be best. I would be fine with a `retryPolicy` scope that controls all retry settings, or something like `retryPolicy.<name>.*` to control retry settings for specific

Reply:
@jorgee I can finish this one so that you can focus on AWS SDK v2